src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -157,8 +138,8 @@ def fetch_historical_data(alert_rule: AlertRule, snuba_query: SnubaQuery) -> Snu
         # DATSET_OPTIONS expects the name 'errors'
         dataset_label = "errors"
     dataset = get_dataset(dataset_label)
-    project = alert_rule.projects.get()
```

Comment:
projects get added to the join table `AlertRuleProjects` _after_ `send_historical_data_to_seer` so I just pass it in instead. In case it fails I didn't want to have to delete more stuff.